addon/templates/components/frost-autocomplete.hbs

```diff
@@ -0,0 +1,38 @@
+{{! Template for the frost-select component }}
+{{frost-text
+  class='frost-autocomplete-text'
+  ariaActiveDescendant=ariaActiveDescendant
+  ariaAutocomplete='list'
+  ariaExpanded=true
+  ariaMultiselectable=false
+  ariaOwns='frost-select-list'
+  hook=(concat hook '-text')
```

Comment:
Change this to `hook=(concat hook 'autocomplete-text')`.  `frost-select` needs to have a similar change made as well so the sub-elements can have unique name scope even though similar elements and naming scheme.

Reply:
I'll go with '-autocompleteText' so it'll like this myAutoComplete-autocompleteText-input for the input